packages/transactions/src/clarity/serialize.ts

```diff
@@ -43,12 +44,14 @@ function serializeBufferCV(cv: BufferCV): Buffer {
 }
 
 function serializeIntCV(cv: IntCV): Buffer {
-  const buffer = cv.value.toTwos(CLARITY_INT_SIZE).toArrayLike(Buffer, 'be', CLARITY_INT_BYTE_SIZE);
+  const buffer = new BN(cv.value.toString())
+    .toTwos(CLARITY_INT_SIZE)
+    .toArrayLike(Buffer, 'be', CLARITY_INT_BYTE_SIZE);
```

Comment:
Is bn.js still the preferred way to do this conversion?

Reply:
I have some code to do it natively with `bigint` from a while back, but I'm not 100% confident in it. Needs a much of tests:
```ts
export function toTwos(value: bigint) {
  // make sure its in range given the number of bits
  if (value < -(1n << (128n - 1n)) || value > (1n << (128n - 1n)) - 1n)
    throw 'Integer out of range given 128 bits to represent.';

  // if positive, return the positive value
  if (value >= 0n) return value;

  // if negative, convert to twos complement representation
  return ~((-value - 1n) | ~((1n << 128n) - 1n));
}

export function fromTwos(value: bigint) {
  if ((value & (1n << (128n - 1n))) > 0n) {
    value = value - (1n << 128n);
  }
  return value;
}
```
Maybe a good task for a separate issue?